components/product/ProductImages/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { ImageObject } from "$live/std/commerce/types.ts";
+import Image from "$live/std/ui/components/Image.tsx";
+import { useId, useState } from "preact/hooks";
+import { ProductImageZoom } from "./ProductImageZoom.tsx";
+import { Thumbnails } from "./Thumbnails.tsx";
+
+export type Props = {
+  image: ImageObject[];
+  width: number;
+  height: number;
+  thumbWidth: number;
+  thumbHeight: number;
+};
+
+export function ProductImages(
```

Comment:
By doing this:
```suggestion
export default function ProductImages(
```

the island can be somehting like this:
```ts
export {default} from "$store/components/product/ProductImages/index.ts"
export type {Props} from "$store/components/product/ProductImages/index.ts"
```